core/src/stored/status.cc

```diff
@@ -648,14 +640,13 @@ static void ListRunningJobs(StatusPacket* sp)
   JobControlRecord* jcr;
   DeviceControlRecord *dcr, *rdcr;
   bool found = false;
-  time_t now = time(NULL);
   PoolMem msg(PM_MESSAGE);
-  int len, avebps, bps, sec;
+  int len;
   char JobName[MAX_NAME_LENGTH];
   char b1[50], b2[50], b3[50], b4[50];
 
   if (!sp->api) {
-    len = Mmsg(msg, _("\nRunning Jobs:\n"));
+    len = Mmsg(msg, _("\nExisting Jobs:\n\n"));
```

Comment:
I think you can leave this as 'Running'.  'Existing' sounds weird.

Reply:
I agree that `Existing` is a bit weird too. But the problem is that `Running` is misleading, since some of the displayed jobs are not always really in a running state, some might be blocked, some others might have just been created but not started yet. 

The point of `Existing` is to say that the job exists in the SD regardless of its status.

I've changed it to `Job Inventory` but feel free to suggest another word you see more fitting.